### DISCUSSION TITLE
Welcome to radiator Discussions!

### DISCUSSION THREAD

**Discussion by thierrygosselin** [thierrygosselin/radiator#104] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by iviceron** [comment]
Hi!
I am trying to use radiator to convert a VCF file to genepop. This data doesn't have a reference genome. Therefore, it is a _the_ novo_ SNP calling. But, using the radiator, I am having this error:
Error in SeqArray::seqGetData(gdsfile = data, var.name = "$ref") : 
  The GDS node "$ref" does not exist.

I am assuming the error means that radiator needs a reference genome. If so, how can I solve it?  Thanks!

**Reply by dawood2014** [reply]
I'm having the same issue. Did you ever get a response that helped solve the issue?

**Reply by thierrygosselin** [reply]
reference genome not required
try again with the latest release, some bug introduced in recent release were corrected

if you're still having a problem, [follow guidelines to open an issue](https://github.com/thierrygosselin/radiator/issues/new?assignees=&labels=&template=bug_report.md&title=) in order to make the error reproducible on my end.

**Reply by Sapicq** [reply]
Hello,

I'am having also the same kind of issue... I am trying to use the fonction tidy_genomic_data in radiator to convert a VCF file to a tidy.vcf format. I had already successfully converted the same vcf file with this function in a previous version of radiator (v1.1.5).

Installation
Windows Server 2012 R2 Standard
R version 4.2.1
radiator_1.2.2
SeqArray_1.36.3 

Code 
>library(radiator)
>tidy.vcf <- tidy_genomic_data(data = "HapMap_MD_S80I25_MAF0.01_He0.6_hwe_SnpD_IndD_1SNP_LD_1156_Assigner_20200501.vcf", strata = "Pop_assigner_19g_updtae_20220915.txt")
################################################################################
######################### radiator::tidy_genomic_data ##########################
################################################################################
Execution date@time: 20220922@1858
Folder created: -8_radiator_tidy_genomic_20220922@1858
Function call and arguments stored in: radiator_tidy_genomic_data_args_20220922@1858.tsv
Analyzing strata file              
    Number of strata: 19
    Number of individuals: 1156
Importing and tidying the VCF...
Execution date@time: 20220922@1858

Reading VCF... 

Data summary: 
    number of samples: 1156
    number of markers: 2340
Error in SeqArray::seqGetData(gdsfile = data, var.name = "$ref") : 
  The GDS node "$ref" does not exist.

Computation time, overall: 18 sec
############################## completed tidy_vcf ##############################

Computation time, overall: 18 sec
######################### completed tidy_genomic_data ##########################


Interestingly when I used a subset of my dataset (only 116 SNPs with 1156 individuals), I have another message error:


> tidy.vcf <- tidy_genomic_data(data = "/Users/Admin/Documents/HapMap_1156_subset_Assigner_20220922.vcf", strata = "/Users/Admin/Documents/Pop_assigner_19g_updtae_20220915.txt")
################################################################################
######################### radiator::tidy_genomic_data ##########################
################################################################################
Execution date@time: 20220922@2008
Folder created: 00_radiator_tidy_genomic_20220922@2008
Function call and arguments stored in: radiator_tidy_genomic_data_args_20220922@2008.tsv
Analyzing strata file
    Number of strata: 19
    Number of individuals: 1156
Importing and tidying the VCF...
Execution date@time: 20220922@2008

Reading VCF... 

Data summary: 
    number of samples: 1156
    number of markers: 116
                                                                    
Filter monomorphic markers
Number of individuals / strata / chrom / locus / SNP:
    Blacklisted: 0 / 0 / 0 / 0 / 0
Error in SeqArray::seqMissing(gdsfile = x, per.variant = TRUE, .progress = FALSE,  : 
  unused argument (.progress = FALSE)
In addition: Warning message:
In as.POSIXlt.POSIXct(x, tz) : unable to identify current timezone 'C':
please set environment variable 'TZ'

Computation time, overall: 2 sec
############################## completed tidy_vcf ##############################

Computation time, overall: 2 sec
######################### completed tidy_genomic_data ##########################

Fichiers
[file.zip](https://github.com/thierrygosselin/radiator/files/9629190/file.zip)

Thanks for the help !

**Comment by tafarakundai** [comment]
Hello, 

I had radiator package before and it was working, but when I got a new computer I am failing to install radiator. The error is related to authorisation token issue, I tried generating one and it is not working.
Using GitHub PAT from the git credential store.
Error: Failed to install 'unknown package' from GitHub:
  HTTP error 401.
  Bad credentials

  Rate limit remaining: 47/60
  Rate limit reset at: 2024-03-21 20:13:19 UTC


May you kindly help with this issue
